docs/quickstarts/sso-quickstart.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Single sign-on (SSO) quick start
+description: Use the Yeoman generator to build a Node.js Office Add-in that uses single sign-on.
+ms.date: 09/07/2022
+ms.prod: non-product-specific
+ms.localizationpriority: high
+---
+
+# Single sign-on (SSO) quick start
+
+In this article, you'll use the Yeoman generator for Office Add-ins to create an Office Add-in for Excel, Outlook, Word, or PowerPoint that uses single sign-on (SSO).
+
+> [!NOTE]
+> The SSO template provided by the Yeoman generator for Office Add-ins only runs on localhost and cannot be deployed. If you are building a new Office Add-in with SSO for production purposes, follow the instructions in [Create a Node.js Office Add-in that uses single sign-on](../develop/create-sso-office-add-ins-nodejs.md).
+
+## Prerequisites
+
+- [Node.js](https://nodejs.org) (the latest [LTS](https://nodejs.org/about/releases) version).
+
+- The latest version of [Yeoman](https://github.com/yeoman/yo) and the [Yeoman generator for Office Add-ins](../develop/yeoman-generator-overview.md). To install these tools globally, run the following command via the command prompt.
+
+    ```command&nbsp;line
+    npm install -g yo generator-office
+    ```
+
+    [!include[note to update Yeoman generator](../includes/note-yeoman-generator-update.md)]
+
+- If you're using a Mac and don't have the Azure CLI installed on your machine, you must install [Homebrew](https://brew.sh/). The SSO configuration script that you'll run during this quick start will use Homebrew to install the Azure CLI, and will then use the Azure CLI to configure SSO within Azure.
+
+## Create the add-in project
+
+> [!TIP]
+> The Yeoman generator can create an SSO-enabled Office Add-in for Excel, Outlook, Word, or PowerPoint, and can be created with script type of JavaScript or TypeScript. The following instructions specify `JavaScript` and `Excel`, but you should choose the script type and Office client application that best suits your scenario.
+
+[!include[Yeoman generator create project guidance](../includes/yo-office-command-guidance.md)]
+
+- **Choose a project type:** `Office Add-in Task Pane project supporting single sign-on (localhost)`
+- **Choose a script type:** `Javascript`
+- **What do you want to name your add-in?** `My SSO Office Add-in`
+- **Which Office client application would you like to support?** `Excel`
```

Comment:
This tells people to use Excel, but quick start includes instructions for all of WXPO. Not necessarily a problem, but consider mentioning up here that the user can pick anything.

Reply:
Fixed.